### PR TITLE
post-media has too small sizes for video files

### PR DESCRIPTION
**Pull request by Quix0r**
Sizes increased for:
- video files that have sizes var being int allowes
- URLs that might be longer (e.g. magnet: URLs) than varbinary(511) allows

As pointed out in https://github.com/friendica/friendica/issues/10193#issuecomment-1158120448=

### CODE REVIEW

**Comment by Quix0r** [comment]
````
+ git update-index --refresh
database.sql: needs update
doc/database/db_post-media.md: needs update 
````

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
See https://friendica.xyz/display/adf174d5-1362-aefa-13e7-5e8666639804, are you working on a fix @Quix0r or should I revert your changes?

**Comment by Quix0r** [comment]
No, you can leave it better there.

File: static/dbstructure.config.php [hunk 1]
```diff
@@ -1252,12 +1252,12 @@
 		"fields" => [
 			"id" => ["type" => "int unsigned", "not null" => "1", "extra" => "auto_increment", "primary" => "1", "comment" => "sequential ID"],
 			"uri-id" => ["type" => "int unsigned", "not null" => "1", "foreign" => ["item-uri" => "id"], "comment" => "Id of the item-uri table entry that contains the item uri"],
-			"url" => ["type" => "varbinary(511)", "not null" => "1", "comment" => "Media URL"],
+			"url" => ["type" => "text", "not null" => "1", "comment" => "Media URL"],
```
Comment (annando) [inline static/dbstructure.config.php:1255]: You can't use text here. The `url` field is a `unique` index.

**Pull request merged**